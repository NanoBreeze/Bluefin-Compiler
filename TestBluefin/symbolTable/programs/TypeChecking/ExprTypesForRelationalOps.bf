int main(float a) {
	float b;
	int c;
	int d;
	int f;
	
	a > c;
	b >= c;
	a < b;
	c > b;
}