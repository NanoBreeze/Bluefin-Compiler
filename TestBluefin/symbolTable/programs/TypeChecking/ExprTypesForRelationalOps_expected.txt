a - t_float - t_float
c - t_int - t_float
a>c - t_bool - t_bool
b - t_float - t_float
c - t_int - t_float
b>=c - t_bool - t_bool
a - t_float - t_float
b - t_float - t_float
a<b - t_bool - t_bool
c - t_int - t_float
b - t_float - t_float
c>b - t_bool - t_bool
